package.json

```diff
@@ -36,66 +36,97 @@
       "name": "Michael Prentice",
       "email": "michael@xlts.dev",
       "url": "https://github.com/splaktar"
+    },
+    {
+      "name": "Jaroslaw Zolnowski",
+      "email": "jarek@xlts.dev",
+      "url": "https://github.com/jzolnowski"
     }
   ],
-  "main": "index.js",
+  "main": "main.js",
   "scripts": {
-    "start": "webpack-dev-server --open --mode development",
-    "test": "karma start karma.config.js",
-    "build": "webpack --mode production",
-    "prettier": "prettier --write \"**/*.{js,json,css,scss,less,md,ts,html}\"",
+    "pre-commit": "lint-staged && npm run lint",
+    "build:templates": "gulp templates",
+    "start:server": "webpack-dev-server --open --mode development --config config/webpack.dev.mjs",
+    "watch:templates": "gulp watch",
+    "start": "npm run build:templates && run-p start:server watch:templates",
     "lint": "eslint .",
-    "update": "npx npm-check -u",
-    "prepare": "husky install"
+    "test:unit": "karma start karma.config.js",
+    "test:unit:watch": "npm run build:templates && run-p test:unit watch:templates",
+    "test:unit:single": "npm run build:templates && karma start karma.config.js --single-run",
```

Comment:
Now that the names of the scripts have changed, we also nee d to update `README.md` accordingly.